b2share/modules/deposit/mappings/deposits/deposits.json

```diff
@@ -101,13 +114,28 @@
             "type" : "string",
             "index": "not_analyzed"
           },
-          "resource_type" : {
-            "type" : "string"
+          "resource_types" : {
+            "properties" : {
+              "resource_type" : {
+                "index":"analyzed",
+                "type" : "string",
+                "analyzer":"english"
+              },
+              "resource_type_general" : {
+                "index":"analyzed",
```

Comment:
`resource_type_general` should be `not_analyzed` as the values come from an enum.